### DISCUSSION TITLE
节点经常无法同步，找不到原因的丢失节点

### DISCUSSION THREAD

**Discussion by MORNLONG** [Chia-Network/chia-blockchain#7018] [Support]
作过端口转发的设置，用外网扫描8444端口是打开的状态，然而6月下旬以来，经常无故的掉节点最终无法同步，严重应影响收割的效率！通过KEVA.APP添加节点和用VPS自建节点的方法都无济于事，手动添加节点的时候一闪就没了。VPN也无法奏效。莫名其妙，过去的两个月基本不怎么出现这个问题，现在每天要断线好几次，无法正常运转了。网络：联通500M。HELP！Chia版本：Linux-GUI

**Comment by loppefaaret** [comment]
please try and verify that you still have 8444 opened ? perhaps try and reboot ? and are you having resources free on the machine that is farming ?

**Reply by MORNLONG** [reply]
Thanks, My problem was solved by a instead of CLI client. So I suggest buddys to use CLI if had a command skill level.

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
